pkg/obc/provisioner.go

```diff
@@ -798,3 +799,17 @@ func (r *BucketRequest) updateReplicationPolicy(ob *nbv1.ObjectBucket) error {
 	log.Infof("updateReplicationPolicy: updated replication successfully")
 	return nil
 }
+
+func getBucketClassNamespace(ob *nbv1.ObjectBucketClaim, bucketOptions *obAPI.BucketOptions, provisionerNS string) string {
+	if ob != nil && ob.Spec.AdditionalConfig["bucketclassNamespace"] != "" {
+		return ob.Spec.AdditionalConfig["bucketclassNamespace"]
+	}
+
+	if bucketOptions != nil && bucketOptions.Parameters["bucketclassNamespace"] != "" {
```

Comment:
Can you please write a comment about what is this `bucketOptions.Parameters["bucketclassNamespace"]` ?

Reply:
@liranmauda, these parameters are available in `StorageClass`. My understanding maybe limited here but in OBCs, we can instead of specifying everything can just specify the `StorageClass`. In that case, details are fetched from `StorageClass` (https://github.com/rook/rook/blob/master/Documentation/ceph-object-bucket-claim.md?plain=1#L105). This is where the `bucketOptions.Parameters` is used.